frontend/src/helpers/helpers.ts

```diff
@@ -216,15 +216,16 @@ export function searchAnatomicalEntities(
 
 export function getForwardConnectionText(connections: any[]) {
   const BASE_FORWARD_CONNECTION_TEXT = "This neuron population connects to connectivity statement with id:";
-  let text = ''
-  if (connections.length) {
-    text += BASE_FORWARD_CONNECTION_TEXT
+  let text = BASE_FORWARD_CONNECTION_TEXT
+  if (connections.length > 1) {
     connections.slice(0, connections.length - 1).forEach((connection: any) => {
-      text = text.concat(` <a style="text-decoration: none" href="/statement/${connection.id}">${connection.id}</a>,`);
+      text = text.concat(` <a style="color:#0000ee" href="/statement/${connection.id}">${connection.id}</a>,`);
     });
     text = text.slice(0, -1);  // remove the last comma
     text = text.concat(" and");
-    text = text.concat(` <a style="text-decoration: none" href="/statement/${connections[connections.length - 1].id}">${connections[connections.length - 1].id}</a>.`);
+    text = text.concat(` <a style="color:#0000ee" href="/statement/${connections[connections.length - 1].id}">${connections[connections.length - 1].id}</a>.`);
+  } else if (connections.length === 1) {
```

Comment:
suggestion: @D-GopalKrishna there is a shorter way to do this... Hint: start slicing at index 1 instead of 0 for the loop. I believe the if then else can be removed completely